src/Components/Navbar/Navbar.tsx

```diff
@@ -1,40 +1,51 @@
-import React from "react";
-import { Link } from "react-router-dom";
-import paths from '@paths/paths';
-import Logo from '@images/XriiLogo/gradient_logo.svg';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import paths from '../../Paths/paths';
+import Logo from '@/images/XriiLogo/gradient_logo.svg';
+import Burger from '@/images/Homepage/burgerMenu.svg';
+import './navbar.scss';
 
+const menuLinksConfigs = [
```

Comment:
it is good practice to add typescript types to the variables you use, so also here

Reply:
Changed for this.
type MenuLinkConfig = {
  path: string;
  name: string;
};

const menuLinksConfigs: MenuLinkConfig[] = [
  { path: paths.HOME, name: 'Home' },
  { path: paths.CASE, name: 'Case Studies' },
  { path: paths.PRICE, name: 'Pricing' },
  { path: paths.ABOUT, name: 'About' },
  { path: paths.BLOG, name: 'Blog' },
];